mne/preprocessing/maxwell.py

```diff
@@ -321,6 +321,10 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
         sss_ctc = _read_ctc(cross_talk)
         ctc_chs = sss_ctc['proj_items_chs']
         meg_ch_names = [info['ch_names'][p] for p in meg_picks]
+        # checking for extra space ambiguity in channel names
+        # between old and new fif files
+        if meg_ch_names[0][3] != ctc_chs[0][3]:
+            ctc_chs = _clean_names(ctc_chs, remove_whitespace=True)
```

Comment:
I also think, there will be more errors later on, I think this is a unique problem might only be at Martinos MEG.

I agree,  `if meg_ch_names[0] not in ctc_chs`  is a better check.

Should I update these changes, and sent another PR ?
Sorry I am still learning GIT and Python.